WooCommerce/Classes/ViewRelated/Dashboard/Factories/StatsDataTextFormatter.swift

```diff
@@ -82,6 +122,33 @@ struct StatsDataTextFormatter {
             return Constants.placeholderText
         }
     }
+}
+
+extension StatsDataTextFormatter {
+
+    // MARK: Delta Calculations
+
+    /// Creates the text showing the percent change from the previous `Decimal` value to the current `Decimal` value
+    ///
+    static func createDeltaText(from previousValue: Decimal, to currentValue: Decimal, locale: Locale = .current) -> String {
+        let numberFormatter = NumberFormatter()
+        numberFormatter.numberStyle = .percent
+        numberFormatter.positivePrefix = numberFormatter.plusSign
+        numberFormatter.locale = locale
```

Comment:
nit: can we easily cache/store configured NumberFormatter? I know DateFormatters are costly to recreate.

Reply:
Good thought! I've moved it in 09bf974 to a computed `static let` property so it only needs to be configured once. (I also removed the `Locale` parameter, which was there just for setting the locale in unit testing — I realized we set the `en_US` locale for our all unit tests in the test plan so we don't need to handle it explicitly here.)